### PR TITLE
[ERP-2110 & ERP-2111] Replaced all remaining Alert and Text Components

### PR DESCRIPTION
**Pull request by NikuPAN**
Replaced all remaining Alert and Text Components to our helper components

### CODE REVIEW

File: src/components/tool/guide/Instructions/ConnectInstructions.js [hunk 1]
```diff
@@ -1,56 +1,78 @@
...
 import InstructionHeading from "./components/InstructionHeading";
 import InstructionText from "./components/InstructionText";
 import { OperatingSystemTabs } from "./components/OperatingSystemTabs";
 
 function LyraInstructions({ username }) {
+  const enableSSHText = () => {
+    return (
+      <TextWithLink
+        textBeforeLink={"You may need to enable the "}
+        link={{
```
[the remark this answers is not in the document]
Comment (NikuPAN) [reply src/components/tool/guide/Instructions/ConnectInstructions.js:15]: Yes, we can use `blue.500`

File: src/components/tool/guide/Instructions/ConnectInstructions.js [hunk 2]
```diff
@@ -1,56 +1,78 @@
...
 
+import TextWithLink from "../../../navigation/TextWithLink";
 import CopyBox from "../../../output/CopyBox";
+import AlertHelper from "../Config/AlertHelper";
 import InstructionHeading from "./components/InstructionHeading";
 import InstructionText from "./components/InstructionText";
 import { OperatingSystemTabs } from "./components/OperatingSystemTabs";
 
 function LyraInstructions({ username }) {
+  const enableSSHText = () => {
```
[the remark this answers is not in the document]
Comment (NikuPAN) [reply src/components/tool/guide/Instructions/ConnectInstructions.js:11]: Thought about that, is that necessary? Would that be too much for a single instance?

[the remark this answers is not in the document]
Comment (NikuPAN) [reply src/components/tool/guide/Instructions/ConnectInstructions.js:11]: The component itself already supporting passing message prop as a function or simple string, we can just add a children prop to support TextWithLink component being wrapped by AlertHelper in this case.

File: src/components/tool/guide/Instructions/ConnectInstructions.js [hunk 3]
```diff
@@ -46,9 +32,21 @@ function LyraInstructions({ username }) {
         <Box>
           <AlertHelper
             alertType={"info"}
-            alertMsg={enableSSHText}
+            // alertMsg={enableSSHText}
```
Comment (ScriptSmith) [inline src/components/tool/guide/Instructions/ConnectInstructions.js:35]: Missed

File: src/components/tool/guide/Config/AlertHelper.js [hunk 4]
```diff
@@ -5,6 +5,7 @@ export default function AlertHelper({
   alertType = "info",
   alertMsg = null,
   onClose = {},
+  children = null,
```
Comment (ScriptSmith) [inline src/components/tool/guide/Config/AlertHelper.js:8]: I'm not really a fan of having two different props to set the same thing, as you have to know internal details of the component in order to know which prop has precedent. Is there a reason to not use one?

**Review by ScriptSmith — CHANGES_REQUESTED** [review]

File: src/components/tool/guide/Config/AlertHelper.js (hunk 4 above)
Comment (NikuPAN) [reply src/components/tool/guide/Config/AlertHelper.js:8]: I'm not convinced to have it to support having to modify the component to support child component. What's the point of having if we already have a prop that already supports message as a function or a string? Should we just keep `children` prop in here?

Comment (NikuPAN) [reply src/components/tool/guide/Config/AlertHelper.js:8]: Actually we can just make one props to support multiple types.

File: src/components/tool/guide/Instructions/ConnectInstructions.js (hunk 3 above)
Comment (NikuPAN) [reply src/components/tool/guide/Instructions/ConnectInstructions.js]: Fixed

File: src/components/tool/guide/Config/AlertHelper.js (hunk 4 above)
Comment (NikuPAN) [reply src/components/tool/guide/Config/AlertHelper.js:9]: Fixed

**Review by ScriptSmith — APPROVED** [review]

**Pull request merged**